dotcom-rendering/src/components/CardHeadline.stories.tsx

```diff
@@ -218,6 +218,40 @@ liveStory.decorators = [
 	]),
 ];
 
+export const labsHeaderStyle: StoryObj = ({ format }: StoryProps) => (
+	<>
+		<Section fullWidth={true} showTopBorder={false} showSideBorders={false}>
+			<h4>Labs redesign on</h4>
+			<CardHeadline
+				headlineText="This is how a card headline in labs looks"
+				format={format}
+				kickerText="Labs"
+				showLabsRedesign={true}
+			/>
+		</Section>
+
+		<Section fullWidth={true} showTopBorder={false} showSideBorders={false}>
+			<h4>Labs redesign off</h4>
+			<CardHeadline
+				headlineText="This is how a card headline in labs looks"
+				format={format}
+				kickerText="Labs"
+				showLabsRedesign={false}
```

Comment:
Should we be integrating the redesign into an existing component or creating a new component?

I think it would be cleaner in future once we bin the old design by removing the files, rather than refactoring these files with these props

Reply:
@dskamiotis I was initially thinking to have a separate story so happy with that compromise. At the same time I would rename the `showLabsRedesign` prop as redesign will lose it's context as time goes on.

@cemms1 if we can't bin it, can't we just create a new component and use it within the new labs component? I don't think it's sustainable to have switches in every redesigned component

Reply:
@cemms1 thanks for the context. It's definitely much easier to copy and paste all the code in the Card component into their own isolated folder, and using a feature switch at the layout level, rather than component level and mixing code for the new design with the old, but given we've started this way it makes sense to continue